.github/workflows/compile.yml

```diff
@@ -11,21 +11,20 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v5
         with:
           submodules: 'recursive'
 
       - name: Setup Typst
-        uses: yusancky/setup-typst@v2.0.1
-        id: setup-typst
+        uses: typst-community/setup-typst@v4
         with:
-          version: 'latest'
+          typst-version: 'latest'
```

Comment:
Hello. Recent versions of Typst have introduced several breaking changes. You may need to approve the workflow to verify whether the components used in your Typst files are still supported in the latest version of Typst. If they are not supported, you can either specify the Typst version here or modify the Typst files.